lib/core/migrate/migrate.js

```diff
@@ -382,6 +382,7 @@ const migrations = [
 			await Migrators.moveOption('showImages', 'display Oddshot', 'showImages', 'display_oddshot');
 			await Migrators.moveOption('showImages', 'display Miiverse', 'showImages', 'display_miiverse');
 			await Migrators.moveOption('showImages', 'display swirl', 'showImages', 'display_swirl');
+			await Migrators.moveOption('showImages', 'display redgifs', 'showImages', 'display_redgifs');
```

Comment:
I could be mistaken, as it's been a while since I've looked at anything with the migrations, but I don't think this line is necessary.

Reply:
Yeah we can remove the migration as its a new host.

Reply:
@honestbleeps @BenMcGarry thanks, removed the line. Does the rest look good?